passkit_ui/lib/src/store_card.dart

```diff
@@ -113,23 +113,20 @@ class StoreCard extends StatelessWidget {
               Align(
                 alignment: Alignment.bottomRight,
                 child: IconButton(
-                  onPressed: () {
-                    showDialog(
-                      context: context,
-                      builder: (_) {
-                        return SimpleDialog(
-                          children: [
-                            for (final entry
-                                in pass.pass.storeCard!.backFields!)
-                              ListTile(
-                                title: Text(entry.label ?? ''),
-                                subtitle: Text(entry.value),
-                              ),
-                          ],
-                        );
-                      },
-                    );
-                  },
+                  onPressed: () => showDialog<void>(
```

Comment:
At some point this could maybe use [`showAdaptiveDialog()`](https://api.flutter.dev/flutter/material/showAdaptiveDialog.html), but that doesn't need to be done in this PR